pdal/util/Utils.hpp

```diff
@@ -194,6 +194,21 @@ namespace Utils
         return (strncmp(prefix.data(), s.data(), prefix.size()) == 0);
     }
 
+    /**
+      Determine if a string ends with a particular postfix.
+
+      \param s  String to check for postfix.
+      \param postfix Postfix to search for.
+      \return  Whether the string ends with the postfix.
+    */
+    inline bool endsWith(const std::string& s, const std::string& postfix)
+    {
+        if (postfix.size() > s.size())
+            return false;
+        return (strncmp(postfix.data(), s.data() + s.size() - postfix.size(),
```

Comment:
You can just strcmp.  The strings are guaranteed to be terminated as of C++11.

Reply:
4e906be